src/App.js

```diff
@@ -0,0 +1,49 @@
+import logo from './logo.svg'
+import React, { useState, useEffect } from 'react'
+import './App.css'
+import Header from './Header'
+import Info from './Info'
+import Languages from './Languages'
+import CategoriesNav from './CategoriesNav'
+import Menu from './Menu'
+import Footer from './Footer'
+
+function App() {
+  const [products, setProducts] = useState([])
+  const [loading, setLoading] = useState(true)
+  const [activeLang, setActiveLang] = useState('pl')
+
+  const makeActiveLang = (lang) => {
+    setActiveLang(lang)
+  }
+
+  const getProducts = async () => {
+    const response = await fetch('./products.json')
```

Comment:
dodaj tutaj try/catch/finally  bo jak będzie jakiś błąd z pobraniem product.json to nie obsłużysz tego. 
setLoading(false) powinno być robione po wykonaniu reqesta również tego który się nie uda. (finally)